src/makeRobotAccountant.js

```diff
@@ -7,6 +7,20 @@
 
 function makeRobotAccountant() {
   // write code here
+  let count = 0;
+  const sumAcc = (a) => {
+    return function(b) {
+      count++;
+
+      if (count > 3 && count % 2 === 0) {
+        return 'Bzzz... Error!';
+      } else {
+        return a + b;
+      }
```

Comment:
```suggestion
      if (count > 3 && count % 2 === 0) {
        return 'Bzzz... Error!';
      }
      
      return a + b;
```
Учитывая что там return - esle можно не писать